test-util/src/main/java/org/jboss/windup/testutil/basics/WindupTestUtilMethods.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.windup.testutil.basics;
+
+import org.jboss.windup.config.DefaultEvaluationContext;
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.RuleProvider;
+import org.jboss.windup.config.RuleSubset;
+import org.jboss.windup.graph.GraphContext;
+import org.ocpsoft.rewrite.config.Configuration;
+import org.ocpsoft.rewrite.param.DefaultParameterValueStore;
+import org.ocpsoft.rewrite.param.ParameterValueStore;
+
+import java.util.List;
+
+/**
+ * A class providing helping static methods for the tests in Windup.
+ */
+public class WindupTestUtilMethods
```

Comment:
I have created this extra class in test-util. I was not able to come up with anything better. This class may aggregate some useful methods like "runOnlyRuleProviders" that will take context and run the rule providers with it.